furusiya/model/maps/generators/dungeon_generator.py

```diff
@@ -4,31 +4,30 @@
 import colors
 from main_interface import Game
 from model.config import config
-from model.item_callbacks import cast_heal, cast_lightning, cast_fireball, cast_confuse
 from model.rect import Rect
-from model.factories import monster_factory, item_factory
-
+from model.maps.generators import map_generator
 
 class DungeonGenerator:
     """
     Generates a multi-room dungeon map in entirety, by mutating map tiles.
     Includes population of monsters, map items, etc.
     """
-    NUM_ITEMS = (10, 20)
-    NUM_MONSTERS = (30, 40)
 
     NUM_ROOMS = (15, 30)
     ROOM_MAX_SIZE = 10
     ROOM_MIN_SIZE = 6
-    NUM_TREES = 1800
-    MAX_ROOM_MONSTERS = 3
-    MAX_ROOM_ITEMS = 2
+    NUM_ITEMS = (10, 20)
+    NUM_MONSTERS = (30, 40)
 
     def __init__(self, area_map):
         self._rooms = []
         self._area_map = area_map
         self._generate_rooms()
-        self._generate_objects()
+
+        map_generator.generate_monsters(self._area_map,
+            random.randint(DungeonGenerator.NUM_ROOMS[0], DungeonGenerator.NUM_ROOMS[1]))
```

Comment:
How about using starred unpacking here? Less clutter.

Reply:
You mean something like `min, max = DungeonGenerator.NUM_ROOMS` or did you mean something else?

Reply:
If you mean `random.randint(*DungeonGenerator.NUM_ROOMS)`, this doesn't seem to run for me. I get a syntax error. :thinking: